qemu/tests/cfg/boot_with_different_vectors.cfg

```diff
@@ -4,20 +4,34 @@
     no RHEL.3 RHEL.4 RHEL.5 RHEL.6
     no RHEL.7.0 RHEL.7.1 RHEL.7.2 RHEL.7.3 RHEL.7.4
     no Host_RHEL.m5, Host_RHEL.m6
-    queues = 4
-    vectors_list = 0 1 2 3 4 5 6 7 8 9 10 11 -1
     type = boot_with_different_vectors
     start_vm = no
     image_snapshot = yes
-    hostpassword = redhat
-    netperf_client = ${vms}
-    netperf_server = localhost
-    netperf_link = netperf-2.6.0.tar.bz2
-    server_path = /var/tmp/
-    client_path = /var/tmp/
-    netperf_test_duration = 300
-    netperf_para_sessions = 6
-    test_protocol = TCP_STREAM
-    enable_msix_vectors = yes
+    pci_nomsi_check_cmd = "cat /proc/cmdline"
     msi_check_cmd = "lspci -vvv -s %s | grep MSI"
-    irq_check_cmd = "cat /proc/interrupts | grep virtio"
+    wo_vectors = yes
+    pci_nomsi = yes
+    variants:
+        - with_vectors:
+            queues = 4
+            vectors_list = 0 1 2 3 4 5 6 7 8 9 10 11 -1
+            hostpassword = redhat
+            netperf_client = ${vms}
+            netperf_server = localhost
+            netperf_link = netperf-2.6.0.tar.bz2
+            server_path = /var/tmp/
+            client_path = /var/tmp/
+            netperf_test_duration = 300
+            netperf_para_sessions = 6
+            test_protocol = TCP_STREAM
+            irq_check_cmd = "cat /proc/interrupts | grep virtio"
+            wo_vectors = no
+            pci_nomsi = no
```

Comment:
I am wondering if this is needed, seems useless when `wo_vectors=no`.

Reply:
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/env_process.py#L1030
We have a parameter `disable_pci_msi`